data/firm_111.csv

```diff
@@ -2,95 +2,95 @@ Ranked Departments,6,Banking & Finance,16083,Africa-wide,Band 4,2,111
 Ranked Departments,354,Corporate/M&A,16083,Africa-wide,Band 3,2,111
 Ranked Departments,467,Dispute Resolution,16083,Africa-wide,Band 4,2,111
 Ranked Departments,996,Projects & Energy,16083,Africa-wide,Band 3,2,111
-Ranked Departments,24,Energy & Natural Resources,16084,Asia-Pacific Region,Recognised Practitioner,2,111
-Ranked Departments,1486,Banking & Finance: Corporate Finance,23,Australia,Band 4,2,111
-Ranked Departments,1483,Banking & Finance: Property Finance,23,Australia,Band 3,2,111
-Ranked Departments,354,Corporate/M&A,23,Australia,Recognised Practitioner,2,111
-Ranked Departments,467,Dispute Resolution,23,Australia,Recognised Practitioner,2,111
-Ranked Departments,24,Energy & Natural Resources,23,Australia,Band 5,2,111
-Ranked Departments,852,General Business Law,25,Azerbaijan,Band 1,2,111
-Ranked Departments,852,General Business Law,30,Barbados,Recognised Practitioner,2,111
+Ranked Departments,24,Energy & Natural Resources,16084,Asia-Pacific Region,Recognised Practitioner,2,9152

```

Comment:
why to keep that file at all?

Reply:
I don't want to keep it. Just can't get rid of it.